artemis-journal/pom.xml

```diff
@@ -15,57 +15,81 @@
+    </parent>
 
-   <artifactId>artemis-journal</artifactId>
-   <packaging>jar</packaging>
-   <name>ActiveMQ Artemis Journal</name>
+    <artifactId>artemis-journal</artifactId>
+    <packaging>jar</packaging>
+    <name>ActiveMQ Artemis Journal</name>
 
-   <properties>
-      <activemq.basedir>${project.basedir}/..</activemq.basedir>
-   </properties>
+    <properties>
+        <activemq.basedir>${project.basedir}/..</activemq.basedir>
+    </properties>
 
-   <dependencies>
-      <dependency>
-         <groupId>org.jboss.logging</groupId>
-         <artifactId>jboss-logging-processor</artifactId>
-         <scope>provided</scope>
-         <optional>true</optional>
-      </dependency>
+    <dependencies>
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging-processor</artifactId>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
 
-      <!--
-          JBoss Logging
-      -->
-      <dependency>
-         <groupId>org.jboss.logging</groupId>
-         <artifactId>jboss-logging</artifactId>
-      </dependency>
-      <dependency>
-         <groupId>org.jboss.logmanager</groupId>
-         <artifactId>jboss-logmanager</artifactId>
-         <scope>test</scope>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-commons</artifactId>
-         <version>${project.version}</version>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-native</artifactId>
-         <version>${project.version}</version>
-      </dependency>
-      <!-- needed to compile the tests -->
-      <dependency>
-         <groupId>junit</groupId>
-         <artifactId>junit</artifactId>
-         <scope>test</scope>
-      </dependency>
-   </dependencies>
-</project>
+        <!--
+            JBoss Logging
+        -->
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.logmanager</groupId>
+            <artifactId>jboss-logmanager</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <!-- needed to compile the tests -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.openhft</groupId>
```

Comment:
Thanls Martyn,
i've read https://activemq.apache.org/artemis/docs/1.2.0/hacking-guide/code.html
and i've to understand one thing:
that link mentions a "test/pom.xml" as the place where i've to put the OpenHFT's deps (they are indeed test deps) with relative licence + artifact's version while you're suggesting to add it in the main pom.xml (not the test's one), thus where is best place to put them?
And about the "extra-test" the procedure is the same as the normal dependencies or there are other things to be aware of?
If 've get it well, for JMH i've to: 
1) add JMH dep on https://github.com/apache/activemq-artemis/blob/master/tests/extra-tests/pom.xml with artifact's version
2) add JMH on https://github.com/apache/activemq-artemis/blob/master/artemis-journal/pom.xml without artifact's version
Have I missed something?
ASAP i'll fix them with your advices :+1: